src/Persistence/Templates/BuiltInTemplates.cs

```diff
@@ -5,18 +5,18 @@ namespace Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
 
 public class BuiltInTemplates
```

Comment:
`class` for holding `const`s and/or `static`s shouldn't be instantiable: 
```suggestion
public static class BuiltInTemplates
```